Relax gstreamer version requirements.

freemangordon opened the pull request:
Use playbin2 "notify::source" signal instead of "signal::source-setup",
which is not available in gstreamer versions prior to 0.10.33.

Signed-off-by: Ivaylo Dimitrov freemangordon@abv.bg

romaxa commented on content/media/gstreamer/GStreamerReader.cpp:
```diff
@@ -88,6 +88,8 @@
 
   if (mPlayBin) {
     gst_app_src_end_of_stream(mSource);
+    if(mSource)
```
if (
 ^ space here

freemangordon replied:
ok, will fix and update pull request

freemangordon replied:
please, close that pull request, I'll create a new one with relevant commits only. Sorry for the noise :)

romaxa commented on configure.in:
```diff
@@ -5759,7 +5759,7 @@ if test "$MOZ_GSTREAMER"; then
     # core/base release number
     # depend on >= 0.10.33 as that's when the playbin2 source-setup signal was
     # introduced
-    GST_VERSION=0.10.33
```
File bug in bugzilla mozilla org with this patch proposal, after that we can push it here

freemangordon committed:
Relax gstreamer version requirements.

Use playbin2 "notify::source" signal instead of "signal::source-setup",
which is not available in gstreamer versions prior to 0.10.33.

Signed-off-by: Ivaylo Dimitrov <freemangordon@abv.bg>

freemangordon committed:
Add default configuration for N900.

Keep in mind this needs CSSU-thumb ScratchBox environment

freemangordon committed:
Enabme WebM, otherwise compilation fails

freemangordon committed:
Some cosmetic changes to "relax gstreamer version requirement"

Signed-off-by: Ivaylo Dimitrov <freemangordon@abv.bg>
